Rakefile

```diff
@@ -101,14 +101,14 @@ end
 namespace :docs do
   desc "Generate markdown docs for Bolt's transport configuration options"
   task :config_reference do
+    options = Bolt::Config::OPTIONS.sort_by { |k, _| k }.to_h
```

Comment:
I think `.sort_by { |k, _| k }` is equivalent to `sort`. Each element is an array and arrays are sorted by length and then by comparing each element in turn. So since the keys must be unique, sorting the hash will be equivalent to sorting by the first element of each array. It's a minor thing though.

Reply:
Oh, nice, it is.